src/main/java/bio/terra/cli/command/app/passthrough/Gcloud.java

```diff
@@ -1,11 +1,44 @@
 package bio.terra.cli.command.app.passthrough;
 
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.exception.SystemException;
+import bio.terra.cli.exception.UserActionableException;
+import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the second-level "terra gcloud" command. */
-@Command(name = "gcloud", description = "Call gcloud in the Terra workspace.")
+@Command(
+    name = "gcloud",
+    description = "Call gcloud in the Terra workspace.",
+    modelTransformer = Gcloud.SubCmdFilter.class)
 public class Gcloud extends ToolCommand {
 
+  static class SubCmdFilter implements CommandLine.IModelTransformer {
+    public CommandLine.Model.CommandSpec transform(CommandLine.Model.CommandSpec commandSpec) {
+      try {
+        if (!Context.getServer().getCloudBuildEnabled()) {
+          commandSpec.removeSubcommand("--gcs-bucket");
+        }
+      } catch (SystemException e) {
+        return commandSpec;
+      }
+      return commandSpec;
+    }
+  }
+
+  private static final Logger logger = LoggerFactory.getLogger(Gcloud.class);
+
+  @CommandLine.Option(
+      names = "--gcs-bucket",
```

Comment:
In PF-2053, Nicole suggested using something different from `--gcs-bucket`, since this is resource name, not bucket name. Let's do `--gcs-bucket-resource`.